tests/opt-testcases/sample.sh

```diff
@@ -27,9 +27,10 @@ run_test    "Sample: ssl_client1, gnutls server, TLS 1.2" \
             -C "error"
 
 requires_protocol_version tls13
+requires_openssl_tls1_3
```

Comment:
Note: this is something that could be done automatically based on seeing `openssl` and `-tls1_3` in the command line. (Perhaps even automagically use `O_NEXT`, though perhaps that would be too much.)

Not insisting on it, as it's pre-existing.

Reply:
I absolutely agree that more autodetection is desirable. But in this PR I want to avoid things that are likely to have an impact on other test cases. So another time.